### PR TITLE
Create code-of-conduct.md

### PR DESCRIPTION
**Pull request by mheadd**
Add code of conduct per the project backlog.

### CODE REVIEW

**Comment by mheadd** [comment]
Not sure I used the right email for you, @bensheldon. Or, if you are the person that should be listed as the contact for CoC violations.

File: code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project lead at bensheldon@codeforamerica.org. All
```
Comment (bensheldon) [inline code-of-conduct.md]: My email is ben@codeforamerica.org

Comment (mheadd) [reply code-of-conduct.md]: Been a a while. I had forgotten the address convention at CfA. Change made.

**Comment by bensheldon** [comment]
Thank you!

**Pull request merged**